.github/workflows/pip-requirements.yml

```diff
@@ -68,10 +67,42 @@ jobs:
           message_success: |
             The changes to the 'requirements-fixed.txt' have been added to this PR.
             All further changes to any of the requirements file will automatically be applied as long an :$REACT_EMOTE: is present.
-          script_call: scripts/compile-requirements.sh scripts/requirements-fixed.txt
           git_diff_root: ncs/nrf
           diff_path: scripts
           diff_file: requirements-fixed.txt
           reaction_emote: "${{ env.REACT_EMOTE }}"
-          git_user_name: "NordicBuilder"
-          git_user_email: "NordicBuilder@github.com"
+          git_user_name: "Nordic Builder"
+          git_user_email: "ci-help@nordicsemi.no"
+          script_call: |
+            OUT_FILE="nrf/scripts/requirements-fixed.txt"
+            echo "Writing frozen requirements to: $OUT_FILE"
+            echo "Log python version: $(python --version)"
+
+            TOPDIR=$(west topdir)
+            cd $TOPDIR
+
+            source ~/.local/bin/virtualenvwrapper.sh
+            [[ $? != 0 ]] && echo "error sourcing virtualenvwrapper" && exit 1
+
+            rmvirtualenv pip-fixed-venv > /dev/null 2>&1
+            # We need to force the result of the following command.
+            # For no aparent reason it returns 1 with the same output as local.
+            mkvirtualenv pip-fixed-venv  > /dev/null 2>&1 || true
+            workon pip-fixed-venv  > /dev/null 2>&1
+            pip3 install pip-tools > /dev/null 2>&1
+            pip3 install setuptools --upgrade
```

Comment:
I think Jan locked this to 'setuptools==71.1.1' because a newer version of setup tools is breaking something.

Might wanna check if there are more differences with the version of this file in the main branch.

Reply:
Issue is solved in setuptools side.